nsdb-sql/src/test/scala/io/radicalbit/nsdb/sql/parser/AggregationSQLStatementSpec.scala

```diff
@@ -385,6 +415,31 @@ class AggregationSQLStatementSpec extends WordSpec with Matchers {
             )
           ))
       }
+
+      "parse it successfully if an aggregation different from count and sum is provided " in {
```

Comment:
maybe we should rename this test case into something like 

`parse it successfully if an avg aggregation is provided`

Then a test case for an arbitrary not supported aggregation could be added.